src/constants/addresses.ts

```diff
@@ -34,19 +34,6 @@ export const V3_CORE_FACTORY_ADDRESSES: AddressMap = {
   [SupportedChainId.CELO_ALFAJORES]: CELO_V3_CORE_FACTORY_ADDRESSES,
 }
 
-export const V3_ROUTER_ADDRESS: AddressMap = {
-  ...constructSameAddressMap('0xE592427A0AEce92De3Edee1F18E0157C05861564', [
-    SupportedChainId.OPTIMISM,
-    SupportedChainId.OPTIMISTIC_KOVAN,
-    SupportedChainId.ARBITRUM_ONE,
-    SupportedChainId.ARBITRUM_RINKEBY,
-    SupportedChainId.POLYGON,
-    SupportedChainId.POLYGON_MUMBAI,
-  ]),
-  [SupportedChainId.CELO]: CELO_V3_ROUTER_ADDRESS,
```

Comment:
this variable should be renamed `CELO_SWAP_ROUTER_ADDRESS` - i verified that it's SwapRouter02, which is v2 + v3, and even though celo doesn't have v2, we decided for calldata parity it's worthwhile to use the same code